### ISSUE TITLE
Error on Google Chrome

### ISSUE THREAD

**Issue by ekabolotina** [vangware/window-open-promise#3]
When opening promised window on Chrome I got this error
```
A boolean is being passed as a fourth parameter to window.open. This is not used and may cause an exception in a future release.
```

**Comment by lukeshiru** [comment]
Must be a recent change in the API, will fix in the next version. Thanks @ekabolotina!

**Comment by lukeshiru** [comment]
Should be fixed on latest (v `4.1.0`). Thanks for reporting it!
